src/config/mongodb.ts

```
@@ -97,3 +100,78 @@ export function getMongoDB(): Db {
 }
+
+// ── #389: Persistent resume-token helpers ────────────────────────────────────
+//
+// Change stream consumers call `watchWithResumeToken` instead of `collection.watch()`.
+// On each event the token is upserted into `_changeStreamTokens`; on restart the stream
+// reopens from the last stored token so no events are missed or double-replayed.
+
+function getTokenCollection(): Collection<{ _id: string; token: ResumeToken }> {
+  return getMongoDB().collection<{ _id: string; token: ResumeToken }>(
+    RESUME_TOKEN_COLLECTION,
+  );
```

ESLint

[error] Replace `⏎····RESUME_TOKEN_COLLECTION,⏎··` with `RESUME_TOKEN_COLLECTION`

(prettier/prettier)